apiServer/pxone/apiserver.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"github.com/gin-gonic/gin"
+	"github.com/portworx/torpedo/apiServer/pxone/utils"
+	"log"
+)
+
+// We will define all API calls here.
+// Once Gin Server starts, it will initialise all APIs it contains.
+// Future work : To have segregated APIs based on need -> We will have to create multiple main calls for initialising.
+func main() {
+	router := gin.Default()
+	router.DELETE("pxone/deletens/:namespace", utils.DeleteNS)
+	router.POST("pxone/createns", utils.CreateNS)
+	router.POST("pxone/inittorpedo", utils.InitializeDrivers)
```

Comment:
Do you want this api exposed? I think you can just call it by default when the app starts

Reply:
I think it's better to expose and this will be then driven by UI Tests. Not every UI test would want driver functionality from Torpedo actually.